packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts

```diff
@@ -87,6 +87,10 @@ const declarations: CommandDeclaration[] = [
 		name: 'editor.deleteLine',
 		label: () => _('Delete line'),
 	},
+	{
+		name: 'editor.duplicateLine',
+		label: () => _('Duplicate Line'),
```

Comment:
It's a minor thing but for consistency it should be "Duplicate line". We aren't really consistent across the app but as a general rule we should use British capitalisation, which means only the first word is capitalised.

Reply:
fixed.

Reply:
Thanks @tessus

Reply:
Yw. I hope I wasn't stepping on your toes.

Reply:
Not at all, just appreciated.